packages/cli/src/bin/pipcook-job.ts

```diff
@@ -1,40 +1,146 @@
 #!/usr/bin/env node
 
 import program from 'commander';
-import { get } from '../request';
-import { run } from '../pipeline';
-import { route } from '../router';
-import { logger } from '../utils';
-
-const PipelineStatus = [ 'creating', 'running', 'success', 'fail' ];
-
-async function list(): Promise<void> {
-  const jobs = await get(`${route.job}/list`);
-  console.table(jobs.rows.map((item: any) => {
-    item.status = PipelineStatus[item.status];
-    return item;
-  }), [ 'id', 'status', 'evaluatePass', 'createdAt' ]);
+import { join, isAbsolute } from 'path';
+import * as fs from 'fs-extra';
+import tar from 'tar-stream';
+import { createGunzip } from 'zlib';
+import { PipelineStatus } from '@pipcook/pipcook-core';
+import { JobResp } from '@pipcook/sdk';
+import { tunaMirrorURI } from '../config';
+import { logger, initClient } from '../utils';
+
+const PipelineStatusStr = [ 'creating', 'running', 'success', 'fail' ];
```

Comment:
Does this status constant also exist in daemon or core. Is it possible to just maintain one copy of this in core

Reply:
For now, it's used in `cli` only, but I think moved into `core` is better, too.

Reply:
I checked the code again, it has beed defined in `sdk`. If it is used by client only, I think we can export it from `sdk`.

Reply:
sounds good